Packs/Sixgill-Darkfeed/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Darkfeed™ - Current Customer Edition",
```

Comment:
please make sure not to change this line

Reply:
the update-release-notes changes it from TM to it unicode. But I should have checked my commit. Will do next time.